src/import/FSHTank.ts

```diff
@@ -171,6 +172,29 @@ export class FSHTank implements Fishable {
               p.id === item ||
               getUrlFromFshDefinition(p, this.config.canonical) === item
           );
+          if (!result) {
+            result = this.getAllInstances().find(
+              profileInstance =>
+                profileInstance.instanceOf === 'StructureDefinition' &&
+                profileInstance.usage === 'Definition' &&
+                (profileInstance.name === item ||
+                  profileInstance.id === item ||
+                  getUrlFromFshDefinition(profileInstance, this.config.canonical) === item) &&
+                profileInstance.rules.some(
```

Comment:
Should this also require that the `kind` is set to `resource`? Or is that requiring too much? Or could this also end up pulling in instances of datatypes if we don't check `kind`?

Reply:
I specifically avoided that check, since this may be a Profile of `complex-type` such as Quantity.